src/validators/BaseValidator.ts

```diff
@@ -90,6 +86,14 @@ export abstract class BaseValidator<T> {
 		return this.isValidationEnabled;
 	}
 
+	protected get shouldValidatorRunConstraints() {
```

Comment:
This should be named `shouldRunConstraints`, and probably type the return to `boolean` just in case?